### PR TITLE
Update set-password-expiration-policy.md

### LINKED ISSUE #13163: New and Federated domains are not affected by password expiration policy
Customer might think this is a tenant-level policy but that's not how Entra ID works and can be misleading causing unexpected password expirations and blocked users.
Following internal incident (IcM 442034193) I would propose adding the following text under "_Important things you need to know about the password expiration feature_".

This feature sets the password policy expiration for each Managed domain present on the tenant, except Federated domains. If later you add a new domain or convert a domain from Federated to Managed, this domain will not contain the password expiration policy. Whenever you convert or add a new domain, you must re-enable password expiration policy to write the password policy expiration in the new domain.

Thank you.
/Nuno

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d322ca91-2125-7486-887f-7c313f2fbcf9
* Version Independent ID: a139620d-3330-a687-709a-35bb270101e4
* Content: [Set the password expiration policy for your organization - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/manage/set-password-expiration-policy?view=o365-worldwide)
* Content Source: [microsoft-365/admin/manage/set-password-expiration-policy.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/manage/set-password-expiration-policy.md)
* Service: **microsoft-365-business**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
Updated more information on IMportant things to know about the password expiration feature

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/13163

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 PLease copy review

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ Follow the steps below if you want to set user passwords to expire after a speci
   
 People who only use the Outlook app won't be forced to reset their Microsoft 365 password until it expires in the cache. This can be several days after the actual expiration date. There's no workaround for this at the admin level.
 
+This feature sets the password policy expiration for each Managed domain present on the tenant, except Federated domains. If later you add a new domain or convert a domain from Federated to Managed, this domain will not contain the password expiration policy. Whenever you convert or add a new domain, you must re-enable password expiration policy to write the password policy expiration in the new domain.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
This feature sets the password policy expiration for each Managed domain, except for Federated domains, present in the organization. If you later add a new domain or convert a domain from Federated to Managed, this domain will not contain the password expiration policy. Whenever you convert or add a new domain, you must re-enable the password expiration policy to enter the password policy expiration in the new domain.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by kwekuako** [comment]
Updated in other PR: https://learn.microsoft.com/en-us/microsoft-365/admin/manage/set-password-expiration-policy?view=o365-worldwide. Closing.

**Comment by kwekuako** [comment]
Updated in other PR: https://learn.microsoft.com/en-us/microsoft-365/admin/manage/set-password-expiration-policy?view=o365-worldwide.

**Pull request closed without merging**